### ISSUE TITLE
Build fails when installing SDCC (last step)

### ISSUE THREAD

**Issue by DaSalba** [Zal0/gbdk-2020#46]
On Linux Ubuntu 19.10 (64-bit), GBDK-2020 builds correctly, but the last step fails with:

```
Installing SDCC
cp: cannot stat '/usr/bin/{as2gbmap,makebin,packihx,sdar,sdasgb,sdcc,sdcdb,sdcdb{,src}.el,sdcpp,sdldgb,sdnm,sdobjcopy,sdranlib,sz80}': No such file or directory
make: *** [Makefile:213: sdcc-install] Error 1
```

I copied the command and ran it myself, and all those files exist but 4:

```
cp: cannot stat '/usr/bin/sdcdb': No such file or directory
cp: cannot stat '/usr/bin/sdcdb.el': No such file or directory
cp: cannot stat '/usr/bin/sdcdbsrc.el': No such file or directory
cp: cannot stat '/usr/bin/sz80': No such file or directory
```

Also, it seems Make is not correctly interpreting that line (`/usr/bin/{as2gbmap,makebin,packihx,sdar,sdasgb,sdcc,sdcdb,sdcdb{,src}.el,sdcpp,sdldgb,sdnm,sdobjcopy,sdranlib,sz80}`), it is feeding it literally to `cp` without expanding the curly braces to get the individual file names.

**Comment by DaSalba** [comment]
I found the problem(s).

1. Ubuntu 19.10 comes with SDCC version 3.80, which is missing those few files. Manually updating to 4.0 fixed this.
2. Make by default uses `sh` as shell interpreter, and it does not support brace expansion, so that copy command fails. See this:

[https://stackoverflow.com/questions/12838278/using-curly-brackets-for-variable-expansion-in-makefile-doesnt-work](https://stackoverflow.com/questions/12838278/using-curly-brackets-for-variable-expansion-in-makefile-doesnt-work)

The problem was fixed when I added `SHELL:=$(shell which bash)` to the top of the `Makefile`. I think GBDK-2020 should be updated to do this (using `which`is better than hardcoding the value, as it may not always be `/bin/bash`).
